hadoop-tools/hadoop-azure/src/site/markdown/abfs.md

```diff
@@ -665,6 +665,79 @@ performance issues.
 Config `fs.azure.enable.check.access` needs to be set true to enable
  the AzureBlobFileSystem.access().
 
+### <a name="authconfigoptions"></a> Auth Options
+`fs.azure.account.key`: To set the account access key. Access keys can be used to authenticate the requests made from the ABFS driver to the Azure storage account.
+
+`fs.azure.account.keyprovider`: If a key provider class is provided the same will be used to get Storage Account key. Else the Simple key provider will be used which will use the given key from the config.
+
+`fs.azure.shellkeyprovider.script`: ShellDecryptionKeyProvider class invokes an external script that will perform the key decryption. The script path has to be set via this config.
+
+`fs.azure.enable.delegation.token`: To enable delegation token manager. Instantiates the class declared in fs.azure.delegation.token.provider.type and issues tokens from the same.
+
+`fs.azure.delegation.token.provider.type`: In case delegation token manager is enabled the AbfsDelegationTokenManager implementation specified in this config will be used as the AbfsDelegationTokenManager implementation.
+
+`fs.azure.sas.token.provider.type`: If the auth type is AuthType.SAS, instantiates the class declared in fs.azure.sas.token.provider.type and issues tokens from it.
+
+`fs.azure.account.auth.type`: To set the auth type to be used. Possible values:   SharedKey, OAuth, Custom, SAS.
+
+`fs.azure.account.oauth.provider.type`: To set the auth provider class to be used.
+
+`fs.azure.account.oauth2.client.id`: To set the OAuth AAD client id when ClientCredsTokenProvider is used.
+
+`fs.azure.account.oauth2.client.secret`: To set the OAuth AAD client secret when ClientCredsTokenProvider is used.
+
+`fs.azure.account.oauth2.client.endpoint`: To set the OAuth AAD client endpoint when ClientCredsTokenProvider is used.
+
+`fs.azure.account.oauth2.msi.tenant`: To set OAuth MSI tenant id when MSITokenProvider is used.
+
+`fs.azure.account.oauth2.msi.endpoint`: To set OAuth MSI endpoint when MSITokenProvider is used.
+
+`fs.azure.account.oauth2.msi.authority`: To set OAuth MSI authority when MSITokenProvider is used.
+
+`fs.azure.account.oauth2.user.name`: To set username when UserPasswordTokenProvider is used.
+
+`fs.azure.account.oauth2.user.password`: To set password when UserPasswordTokenProvider is used.
+
+`fs.azure.account.oauth2.refresh.token`: To set OAuth refreshtoken when RefreshTokenBasedTokenProvider is used.
+
+`fs.azure.account.oauth2.refresh.token.endpoint`: To set OAuth refresh token end point when RefreshTokenBasedTokenProvider is used.
+
+### <a name="featureconfigoptions"></a> Feature Options
```

Comment:
Maybe provide more context to the feature and then list the relevant configs.

Reply:
Done